sdk/servicebus/service.projects

```diff
@@ -0,0 +1,39 @@
+<!--
+  This file allows overrides and extensions to the build configuration defined
+  by the central engineering system, and is considered during pipeline builds for
+  CI, test runs, and package release.
+
+  Based on the SDKType variable (msbuild property or env variable) this will filter
+  the set of projects to to match what type of SDK we are interested in building.
+
+  SDKType values
+   - empty or "client" will build the new track 2 client/data libraries
+   - "mgmtclient" will build the track 2 mgmt libraries
+   - "data" will build the track 1 data plane libraries
+-->
+
+<Project>
+  <ItemGroup Condition="'$(SDKType)' == '' or '$(SDKType)' == 'client'">
+    <ProjectsToInclude Include="$(MSBuildThisFileDirectory)Azure.Messaging.*\**\*.csproj" />
+    <ProjectsToInclude Include="$(MSBuildThisFileDirectory)Microsoft.Azure.WebJobs.Extensions.ServiceBus\**\*.csproj" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(SDKType)' == 'mgmtclient'">
```

Comment:
@jsquire can you please keep this with the client set for now. I know I split it out with eventhub but I'm chatting with the management team and would like to keep them together until we come up with the plan.

cc @allenjzhang @markcowl @m-nash

Reply:
Actually I just noticed servicebus doesn't yet have a track 2 management library so it is really just a question of whether or not we should exclude this part now or keep it as a placeholder for later. I'd probably remove it and we can add it back when we get a track 2 mgmt library and decide to go this approach.

Reply:
I can pull it from here.   There's no `Azure.ResourceManager` client for Service Bus at this point, so this was just ensuring expected behavior for future use.